.travis.yml

```diff
@@ -3,3 +3,7 @@ node_js:
   - "node"
   - "6"
   - "0.12"
```

Comment:
"Node" is 4. But anyways it's a copy of yours now, minus 0.10 which broke for me.